### ISSUE TITLE
Compute size of text in advance.

### ISSUE THREAD

**Issue by Kesanov** [Dominaezzz/kotlin-imgui#28]
I could not find a way to compute the size of a text in advance, without rendering it on screen. 

As far as I know, ImGui has a `CalcTextSize` function that facilitates this functionality.

**Comment by Dominaezzz** [comment]
Did you not find `ImGui.calcTextSize()`?

**Comment by Josef-Vonasek** [comment]
No. Does it exist?

On a similar note, is there a way to

**Comment by Kesanov** [comment]
Nope. Quick search returns [nothing](https://github.com/Dominaezzz/kotlin-imgui/search?q=calcTextSize).

**Comment by Dominaezzz** [comment]
It's a generated file, so it's not in the repo. You'll have to rely on auto compete, but it does indeed exist.

**Comment by Kesanov** [comment]
Autocomplete doesn't return anything either. I use `import com.imgui.*`.

**Comment by Dominaezzz** [comment]
Odd, I'm fairly certain it should be there. I'll confirm when I get time.
Are you able to Ctrl+Click on any `ImGui` functions? What are the functions near `ImGui.calcTextSize()`, (the functions are sorted)

**Comment by Kesanov** [comment]
```
beginTabBar
beginTabItem
beginTooltip
bullet
bulletText
button
calcItemWidth
calcListClipping
captureKeyboardFromApp
captureMouseFromApp
checkbox
checkboxFlags
```

**Comment by Dominaezzz** [comment]
Ah it returns `Vec2`, which I don't handle yet, so it is indeed missing. I had begun work on this in the `im_vector` branch.

In the meantime you can use the cinterop apis to workaround this.
